docker/sample/zms/create-self-signed-certs.sh

```diff
@@ -8,14 +8,14 @@ cd "$(dirname "$0")"
 echo 'running...'
 
 # create CSR
-CN='Sample Self Signed ZMS' SAN="${ZMS_HOST}" openssl req -nodes \
+CN='Sample Self Signed ZMS' SAN="${ZMS_HOST}" IPSAN="${HOST_EXTERNAL_IP}" openssl req -nodes \
   -config "${SELF_SIGN_CNF_PATH}" \
   -reqexts v3_req -extensions usr_cert \
   -newkey rsa:4096 \
   -keyout "${DEV_ZMS_CERT_KEY_PATH}" \
   -out "${DEV_ZMS_CSR_PATH}" 2> /dev/null
 # sign request
-SAN="${ZMS_HOST}" openssl x509 -req -days 364 \
```

Comment:
@WindzCUHK I changed this value to 364 since chrome enforces maximum expiry on certificates now and it doesnt like 10 year valid certificate, when zms is used from browser for the UI

Reply:
I changed this parameter to 364 days.